src/Command/AccountExpirationNotifyCommand.php

```diff
@@ -0,0 +1,258 @@
+                )
+            );
+        }
+    }
+
+    /**
+     * Get accounts with passwords last modified between $minTimeStamp and $maxTimeStamp.
+     *
+     * @param \DateTime $minTimeStamp The minimum time stamp for password modification.
+     * @param \DateTime $maxTimeStamp The maximum time stamp for password modification.
+     *
+     * @return array Array of Accounts with passwords last modified between $minTimeStamp and $maxTimeStamp.
+     */
+    protected function getAccounts(\DateTime $minTimeStamp, \DateTime $maxTimeStamp)
+    {
+        // Create a query builder.
+        $queryBuilder = $this->entityManager->createQueryBuilder();
+
+        // Build a query to get all accounts with passwords last modified between $minTimeStamp and $maxTimeStamp.
+        $queryBuilder
+            ->select('account')
+            ->from(Account::class, 'account')
+            ->join('account.password', 'password')
+            ->where(
+                $queryBuilder->expr()->between(
+                    'password.modificationTimeStamp',
+                    '?1',
+                    '?2'
+                )
+            )
+            ->setParameter(1, $minTimeStamp)
+            ->setParameter(2, $maxTimeStamp);
+        // Return the query result.
+        return $queryBuilder->getQuery()->getResult();
+    }
+
+    /**
+     * Notify account holders.
+     *
+     * @param array                 $accounts      Array of accounts to notify holders.
+     * @param \Twig\TemplateWrapper $emailTemplate Twig email template to use for notification.
+     *
+     * @return void
+     */
+    protected function notifyAccountHolders(array $accounts, \Twig\TemplateWrapper $emailTemplate)
+    {
+        foreach ($accounts as $account) {
+            // Get the Person this Account belongs to.
+            $person = $account->getPerson();
+
+            // Create a time stamp for when the password has expired or will expire.
+            $expireTimeStamp = $account->getPasswordEntity()->getModificationTimeStamp();
+            $expireTimeStamp->add(new \DateInterval($this->maximumPasswordAge));
+            $expireTimeStamp->setTimeZone(new \DateTimeZone(date_default_timezone_get()));
+
+            $this->output->write('  ' . $account->getUserId());
+            $this->output->write(' (' . $person->getFirstName() . ' ' . $person->getLastName() . ')');
+            $this->output->writeln(' expiration: ' . $expireTimeStamp->format('c'));
+
+            // Construct an array for twig containing data needed for rendering the email template.
+            $twigData = array(
+                'person' => $person,
+                'hostname' => $this->hostName,
+                'expireTimeStamp' => $expireTimeStamp,
+            );
+
+            $mailData['recipient'] = $person;
+
+            $this->mailer->sendEmailMessage(
+                $emailTemplate,
+                $twigData,
+                $person->getEmailAddress()
```

Comment:
This seems to be expecting an array of addresses.

Reply:
I refactored the Mailsender Utility class's method to take an array for the $toAddress, forgot to change this usage to array